dapps/marketplace/src/pages/listings/Listings.js

```diff
@@ -81,6 +81,8 @@ class Listings extends Component {
             <fbt:param name="category">
               {this.state.search.subCategory.type === 'clothingAccessories'
```

Comment:
Just a suggestion. If we are going to override more category names here, We can probably do it with a `switch` statement. IMO, It will be more readable than nesting conditional statements.

Reply:
I don't disagree, but I remembered reading this 😁 https://medium.com/javascript-scene/nested-ternaries-are-great-361bddd0f340 Also, I was being lazy.